### ISSUE TITLE
Pre trainning of the models

### ISSUE THREAD

**Issue by yedide** [kaushaltrivedi/fast-bert#73]
Hi, 
Thanks for your work.
Do you plane to add pre-trainning tools on custom data?
Many thanks!

**Comment by kaushaltrivedi** [comment]
Yes. In fact that’s what I am working on. You can use the fine tuning language model functionality from the master branch. It’s still work in progress with documentation pending.

**Comment by yedide** [comment]
Ok great! I will definitely be interested by this functionality!
Do you know if it's roughly a short, mid or long term project?
Many thanks!

**Comment by kaushaltrivedi** [comment]
It's already in place.  Checkout the README file.  Download the latest version 1.4.

**Comment by djstrong** [comment]
What about pre-training language model (e.g. training BERT masked language model from scratch)?

**Comment by apohllo** [comment]
I looked into Readme, but I cannot find a version of the learner that would not require the pretrained model path. There is only a section of fine-tuning. Can you please explain how the class should be used when training from scratch?
